app/general/models.py

```diff
@@ -135,7 +135,14 @@ class Document(models.Model):
     document_type = models.CharField(
         max_length=200, choices=document_type_choices, verbose_name=_("document category")
     )
-    document_data = models.TextField(blank=True, verbose_name=_("document data"))
+    document_data = models.TextField(
+        blank=True,
+        verbose_name=_("Searchable content"),
+        help_text=_(
+            "This is the document's full-text which users can search through. "
+            "By default, this is extracted from the document's PDF (if exists), but it can also be edited."
```

Comment:
My suggestion:
```python
            "The searchable text extracted from the document where possible, but it can also be edited."
```
While currently limited to PDF, we should leave room for other document types (even online ones).

For user facing strings, please remember "typographical" punctuation marks like curly quotes, etc. (I removed it in my suggestion.)